.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+
+# Description
+<!--- Describe your changes to provide context for reviewrs -->
+
+## How to test
+
+<!--- Steps to run to verify the code changes work -->
+
+1.
+1.
+1.
```

Comment:
IMO I see this section as too specific for those PRs that require manual testing. The changes should be accompanied by unit tests that verify the code changes work.